core/src/main/scala/org/bitcoins/core/wallet/signer/Signer.scala

```diff
@@ -124,6 +124,31 @@ object NestedSigning {
     override val scriptPubKeyToSatisfyOpt: Option[ScriptPubKey] = Some(
       redeemScript)
   }
+
+  /** For Locktime signing, this will be passed to the nested Signer.
+    *
+    * @param nestedSPK This should be [[LockTimeScriptPubKey.nestedScriptPubKey]]
+    */
+  case class LockTimeOverrides(nestedSPK: ScriptPubKey)
+      extends OverridesForNestedSigning {
+    override val sigComponentToSignOpt: Option[TxSigComponent] = None
```

Comment:
Why is this `None`?

Reply:
So I've realized that things work in a kinda weird way right now where even on nested calls, we pass in the external parameters `output, unsignedTx, inputIndex`. Thus, unless the nesting is withing a Witness SPK which would then require a `WitnessSigComponent` (as opposed to `Base`) then the `TxSigComponent` is actually constructed correctly.

I did not change this weirdness in the spirit of not changing any control flow I don't have to in my refactors.

I opened #828 as an attempt to make this better and clearer and generally simplify things. Let me know what you think of the approach.

Reply:
#828 has been replaced by #830